package.json

```diff
@@ -62,6 +62,7 @@
     "@storybook/addon-knobs": "^5.3.18",
     "@storybook/addon-storysource": "^5.3.18",
     "@storybook/addon-viewport": "^5.3.17",
+    "@storybook/client-api": "^5.3.18",
```

Comment:
The storybook on master was missing this and won't run without it.

Reply:
What does it do? The package has no readme.

Reply:
Just seeing this locally now too. Odd.